misc/pytorch_toolkit/chest_xray_screening/tests/test_export.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import os
+import json
+from chest_xray_screening.utils.download_weights import download_checkpoint
+from chest_xray_screening.utils.exporter import Exporter, OPENVINO_DIR
+
+
+def _get_config_():
+    path = os.path.dirname(os.path.realpath(__file__))
+    with open(os.path.join(path, 'test_config.json'), ) as f1:
+        config_file = json.load(f1)
+
+    return config_file['export']
```

Comment:
Please, unify this function with similar functions from `test_train` and `test_inference`, move it into some common file and export it from there.